influence_benchmark/stats/utils_pandas.py

```diff
@@ -41,18 +41,28 @@ def group_turns_df_to_traj_df_final(turns_df: pd.DataFrame) -> pd.DataFrame:
     traj_df: Dataframe containing one entry for each traj
     """
     # Get the final turn for each trajectory
-    traj_final_df = (
+    traj_df = (
         turns_df.groupby(["env_name", "initial_state_id", "trajectory_id"])
-        .apply(lambda x: x.loc[x["turn"].idxmax()])
-        .reset_index(drop=True)
+        .agg(
+            {
+                "timestep_reward": lambda x: x.iloc[-1],
+                "timestep_influence_level": lambda x: x.iloc[-1],
```

Comment:
Do we actually also want to change the timestep_influence_level aggregation? That would require also changing the influence prompt, which is something that we're not really keeping track of (or at least, I didn't realize this)

Maybe by default we should keep this be the mean, and we can change it to the final later if we commit to using final reward everywhere?